src/api/v1beta1/dynakube/test/certs_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"testing"
 
```

Comment:
why are these tests in their own `test` folder?

Reply:
These test have been created in this way

Reply:
you can't move them, otherwise there will be a circular dependency ...

Reply:
```
package github.com/Dynatrace/dynatrace-operator/src/api/v1beta1/dynakube
        imports github.com/Dynatrace/dynatrace-operator/src/scheme
        imports github.com/Dynatrace/dynatrace-operator/src/scheme/fake: import cycle not allowed in test
```